crates/loro-internal/src/encoding/fast_snapshot.rs

```diff
@@ -18,22 +18,27 @@ use std::io::{Read, Write};
 use crate::{oplog::ChangeStore, LoroDoc, OpLog, VersionVector};
 use bytes::{Buf, Bytes};
 use loro_common::{IdSpan, LoroError, LoroResult};
+use tracing::trace;
 
 use super::encode_reordered::import_changes_to_oplog;
 
+pub(crate) const EMPTY_MARK: &[u8] = b"E";
```

Comment:
Otherwise, it’s ambiguous. You don't know whether it’s encoded as an empty array (it happens if the latest state has the exact same state as the GC state), or it is a None (which means it’s not encoded on purpose)